module/VuFind/src/VuFind/Auth/LoginTokenManager.php

```diff
@@ -251,18 +266,22 @@ public function requestIsFinished(): void
     /**
      * Create a new login token
      *
-     * @param \VuFind\Db\Row\User $user      user
-     * @param string              $series    login token series
-     * @param string              $sessionId Session identifier
-     * @param int                 $expires   Token expiration timestamp
+     * @param User   $user           User
```

Comment:
Is it worth mentioning in the comment that this also has the side effect of deleting outdated tokens? Now that the functionality has been combined, the method name might be misleading.

Reply:
Renamed, made protected and added a simplified public createToken.